content/tt/paka.md

```diff
@@ -21,39 +21,44 @@ PAKA (roughly _The PACK_) was a babyface tag team, active in [Prime Time Wrestli
+Taras made his in-ring debut in the opener of PTW's first show ever - [REVOLUCJA](@/e/ptw/2021-10-09-ptw-1-revolucja.md) - where he faced and defeated German veteran and ex-[WWE](@/o/wwe.md) talent Axel Tischer. During the build-up for the match he was hailed as the first-ever alumni trained only in [PTW Academy](@/o/ptw-academy.md). Boro competed in a battle royal in a losing effort, while Disco Pablo competed alongside [Santino](@/w/santino.md) in a tag team match against [Syriusz Dziedzic](@/w/dziedzic.md) and Marius Al-Ani, also to no success.
 
-For the first few shows, PAKA focused on establishing themselves as babyfaces and Pawłowski's favourites against heels such as [Sinister](@/w/sinister.md), [Dziedzic](@/w/dziedzic.md) and [Remo](@/w/remo.md).
+For the first few shows PAKA focused on establishing themselves as babyfaces and [Pawłowski's](@/w/pan-pawlowski.md) favourites against heels such as [Sinister](@/w/sinister.md), [Dziedzic](@/w/dziedzic.md) and [Remo](@/w/remo.md).
 
 ### Feud wih Bad Bones
 
-Later on Taras - alongside his partners from PAKA - got involved in a feud against [Bad Bones](bad-bones.md), who was then trying to destroy Pawłowski and his federation from the inside. Taras was beaten up and injured after one of the training sessions held at [PTW Performance Center](@/v/ptw-targowa.md) by an annonymous attacker - who, almost two years later, was revealed to be [Longman](@/w/wiktor-longman.md). The feud between Taras and Bad Bones ended when the Polish wrestler was defeated at [PTW x RyuCon 2022](@/e/ptw/2022-07-31-ptw-x-ryucon.md).
-At [PTW Underground 6](@/e/ptw/2022-06-26-ptw-underground-6.md) Boro competed against [John "Bad Bones" Klinger](@/w/bad-bones.md), who was feuding with Pawłowski by taking out his allies such as PAKA along the way. During that match Boro suffered a knee injury which was his weak point sincethe MZW days of his career. For the next couple of shows Boro was sidelined as the manager of his PAKA teammates.
+Later on Taras - alongside his partners from PAKA - got involved in a feud against [Bad Bones](bad-bones.md), who was then trying to destroy Pawłowski and his federation from the inside. After one of the training sessions held at [PTW Performance Center](@/v/ptw-targowa.md) Taras was beaten up and injured by an annonymous attacker - who, almost two years later, was revealed to be [Longman](@/w/wiktor-longman.md). The feud between Taras and Bad Bones ended when the Polish wrestler was defeated at [PTW x RyuCon 2022](@/e/ptw/2022-07-31-ptw-x-ryucon.md).
+
+At [PTW Underground 6](@/e/ptw/2022-06-26-ptw-underground-6.md) Boro competed against [John "Bad Bones" Klinger](@/w/bad-bones.md), who was feuding with Pawłowski by taking out his allies such as PAKA along the way. During that match Boro suffered a knee injury, which was his weak point since the MZW days of his career. For the next couple of shows Boro was sidelined as the manager of his PAKA teammates.
```

Comment:
Combine the two - both are about Bones (under various names) feuding with PAKA and Pawłowski; and out of order chronologically.